theano/gpuarray/opt.py

```diff
@@ -2219,7 +2286,7 @@ def local_gpu_svd(op, context_name, inputs, outputs):
                                'conv_gemm',
                                'gpuarray', 'fast_compile', 'fast_run')
 
-
+conv_metaopt.register(abstractconv_groupopt.query(*['+' + name for name in abstractconv_groupopt._names]).opts)
```

Comment:
Yes, I was planning to remove it later when I  modify it to support optimizer_including and optimizer_excluding.